modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

```diff
@@ -0,0 +1,180 @@
+= Starting and Stopping Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, esb, deploy, launch, run, terminate
+
+[NOTE]
+====
+This section deals with deploying to standalone instances of the Mule Runtime. To take full advantage of managing and monitoring features, you can deploy to the same Mule Runtime via the xref:runtime-manager::index.adoc[Runtime Manager]. See xref:runtime-manager::deployment-strategies.adoc[Deployment Strategies] for a deeper look at the different deployment alternatives offered by the platform.
+====
+
+Mule uses the Java Service Wrapper to control the Java Virtual Machine (JVM) from your native OS. The wrapper provides many options and features, including the ability to run Mule as a Unix daemon or install or remove Mule as a Windows Service. The wrapper can handle system signals and start parameters, and overall provides much better interaction between the JVM and the underlying OS.
+
+
+== Startup and Shutdown Script
+
+The wrapper is called by a script in `$MULE_HOME/bin`.
+
+The table below lists all the parameters that the script accepts.
+
+[%header,cols="30a,70a"]
+|===
+|Parameter |Description
+| `start` |Starts the Mule server in the terminal background.
```

Comment:
use Mule or Mule runtime engine. It's never clear what "server" means separate from Mule in these contexts.